es-vue-base/src/lib-components/EsReviewModal.vue

```diff
@@ -46,7 +44,9 @@
                                 height="32px"
                                 class="mx-2"
                                 :rating="avgRating" />
-                            <span class="font-size-base font-weight-normal align-self-center">({{ reviews.length }})</span>
+                            <span class="font-size-base font-weight-normal align-self-center"
+                                >({{ reviews.length }})</span
+                            >
```

Comment:
This looks like prettier and eslint are conflicting locally. That shouldn't be happening, prettier itself shouldn't be formatting anything.

Reply:
Right, that's why I disabled format on save. I'll create a follow-up ticket for this.